.env

```diff
@@ -8,3 +8,10 @@ VANILLA_DOCKER_FASTCGI_BUFFER_SIZE=128k
 VANILLA_DOCKER_FASTCGI_BUFFERS="4 256k"
 VANILLA_DOCKER_FASTCGI_BUSY_BUFFERS_SIZE=256k
 VANILLA_DOCKER_FASTCGI_TEMP_FILE_WRITE_SIZE=256k
+
+# https://docs.imgproxy.net/configuration
+IMGPROXY_IGNORE_SSL_VERIFICATION=true
+#https://docs.imgproxy.net/signing_the_url
+#IMGPROXY_KEY=
+#IMGPROXY_SALT=
+#IMGPROXY_SIGNATURE=
```

Comment:
Are those meant to be used in the future?

Reply:
yes these would be enabled in production for signing the requests, so no external entities could use our imgproxy install if they didn't have the key. They are there so people can see where to put them if they want to test/work with signatures. They do work, but they have to also be added to the config file in Vanilla to generate the signature for the requests.

Reply:
To install in your local docker you can pull the change (this branch) _rebuild your docker_ containers and you will have the new image responding at http://localhost:8600.